### PR TITLE
Readme updates

### PR DESCRIPTION
**Pull request by elliotabbott**
Please accept

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,3 @@
 # gh-demo
-Foo test
+
```
Comment (aabmass) [inline README.md:2]: Nice!

File: README.md [hunk 2]
```diff
@@ -1,2 +1,3 @@
...
+Hello world!3
```
Comment (aabmass) [inline README.md]: pls add a space
```suggestion
Hello world! 3
```

**Review by aabmass — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (elliotabbott) [reply README.md]: Done!

**Review by aabmass — APPROVED** [review]
LGTM

**Pull request merged**